### ISSUE TITLE
Aliases are not listed in help menu

### ISSUE THREAD

**Issue by ethagnawl** [tmuxinator/tmuxinator#420]
As noted by @cwhits in [https://github.com/tmuxinator/tmuxinator/pull/416#issuecomment-22135571](#416), aliases defined using `map` (e.g. `map "edit" => :new`) are not listed by `tmuxinator help`, which appears to be by (Thor's) design. Since it looks like there's no way to achieve this using Thor's `aliases`/`map` API, it might be worth creating explicit descriptions/methods for these commands.

**Comment by J3RN** [comment]
Another, but perhaps relevant, facet of this is that I feel like the 'edit' command should be split off from the 'new' command. I feel that if you try to edit a non-existent project file, Tmuxinator shouldn't just generate a new project file for you. Chief reason: typos. Typing `mux edit prjoect` when trying to edit the 'project' file would create a new project with this hideous name, which then must be removed.

Should I split this off into another issue?

**Comment by cwhits** [comment]
The only mapped commands are `edit` and `open`. If `edit` is broken off into a separate issue to prevent it from creating new projects because of a typo, `open` should be a mapped alias of `edit` instead of `new` for the same reasons.

**Comment by adamstrickland** [comment]
I agree I don't really like the aliased commands.  Makes the code overly complicated as well.

IMO we should table this (#420) issue and create a new one that will streamline the set of commands, as @J3RN suggested.

**Comment by ethagnawl** [comment]
Bumping this issue because it came up again in #498.
